gdk-pixbuf/src/subclass/pixbuf_animation.rs

```diff
@@ -160,8 +160,10 @@ unsafe extern "C" fn animation_get_size<T: PixbufAnimationImpl>(
     }
 }
 
-static STATIC_IMAGE_QUARK: Lazy<glib::Quark> =
-    Lazy::new(|| glib::Quark::from_str("gtk-rs-subclass-static-image"));
+fn static_image_quark() -> glib::Quark {
```

Comment:
I am not really a fan of this change tbh, as at the end of the day we don't get rid of once_cell from our dependencies tree. I would keep such Lazy usages

Reply:
May you explain? This PR literally removes usage of `once_cell` crate.

Reply:
Never mind, I got confused by the tests/examples deps pulling once_cell.

Reply:
I actually don't think you need a function here at all, the quark is used only inside a single function which can define the static itself. See https://github.com/gtk-rs/gtk4-rs/pull/1580/commits/3b8e9b04808509278efb14ead45873e4921cf81b

Reply:
Most of the quarks may be moved inside of a function. But there are a few places where they are used across multiple functions. E. g. `PRINT_HANDLER`.